docs/src/models/atmodel.rst

```diff
@@ -0,0 +1,9 @@
+:mod:`models.atmodel` -- Author-topic models
+======================================================
+
+.. automodule:: gensim.models.atmodel
+    :synopsis: Author-topic models
+    :members:
```

Comment:
I don't really understand how this works, I just made it the same way as the LdaModel rst file. What should I do?